apps/web-mzima-client/src/env.json

```diff
@@ -1,6 +1,6 @@
 {
   "production": true,
-  "backend_url": "https://mzima-dev-api.staging.ush.zone/",
+  "backend_url": "http://localhost:8080/",
```

Comment:
Is this change intentional?

Reply:
Good spot, that was an oversight!

Reply:
Although I think it does makes sense, that is the address the API get when setting it up locally, isn't it? It makes it easier for new contributors to get started.